src/appleseed-max-impl/appleseedobjpropsmod/appleseedobjpropsmod.rc

```diff
@@ -63,8 +63,9 @@ BEGIN
     CONTROL         "Glossy",IDC_BUTTON_VISIBILITY_GLOSSY,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,5,71,94,10
     CONTROL         "Specular",IDC_BUTTON_VISIBILITY_SPECULAR,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,5,82,94,10
     CONTROL         "SSS",IDC_BUTTON_VISIBILITY_SSS,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,5,93,94,10
-    CONTROL         "SSS Set",IDC_SSS_SET,"CustEdit",WS_TABSTOP,43,104,58,10
-    LTEXT           "SSS Set Id:",IDC_SSS_SET_LABEL,5,105,38,8
+    CONTROL         "SSS Set",IDC_SSS_SET,"CustEdit",WS_TABSTOP,43,116,58,10
+    LTEXT           "SSS Set Id:",IDC_SSS_SET_LABEL,5,116,38,8
+    CONTROL         "Generate Assembly",IDC_BUTTON_GENERATE_ASSEMBLY,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,5,104,94,10
```

Comment:
I wonder if we could find a better caption for this... "Optimize for Instancing"?

Reply:
May be "Optimize Instances"?

Reply:
"Optimize for Instancing" seems clearer to me...

Reply:
Done.